lib/os-modules/Taskfile.yaml

```diff
@@ -139,6 +148,63 @@ tasks:
           cd -
         done
 
+  pr:
+    desc: |
+      Create pull requests for the changes pushed to SYNC_BRANCH.
+      Example: `task os:pr -- terraform-spacelift-automation`
+    summary: |
+      This will create pull requests from the SYNC_BRANCH to main for each of the specified
+      Terraform module repositories using GitHub CLI.
+      To create PRs for specific repositories, pass their names as arguments:
+      `task os:pr -- terraform-custom-module`
+      or for multiple modules: `task os:pr -- "terraform-custom-module terraform-another-module"`
+
+    vars:
+      MODULES: "{{if .CLI_ARGS}}{{.CLI_ARGS}}{{else}}{{.DEFAULT_MODULES}}{{end}}"
+      PR_TITLE: "chore: sync with latest template state"
+      PR_BODY: |
+        This PR syncs the repository with the latest state from `terraform-module-template`.
```

Comment:
Can we update this PR body to include the tag / latest commit SHA from terraform-module-template so we can track what version this update is coming from?

Reply:
We should this as a follow up, so no need to add it to this PR! 

![Arc 2025-07-01 13 04 35](https://github.com/user-attachments/assets/ec56a72b-a004-4a9b-b2ab-3167597d3ff5)
![Arc 2025-07-01 13 07 20](https://github.com/user-attachments/assets/14c97260-d4ff-48ab-a26d-f6b196087fd8)